wavesrv/cmd/main-server.go

```diff
@@ -618,6 +619,15 @@
 	}
 }
 
+func HandleConfig(w http.ResponseWriter, r *http.Request) {
+	qvals := r.URL.Query()
+	filePath := qvals.Get("path")
+	log.Printf("URL query: %v %v\n", qvals, filePath)
+	fullPath := path.Join(scbase.GetWaveHomeDir(), "config", filePath)
+	log.Printf("fullPath: %v", fullPath)
+	http.ServeFile(w, r, fullPath)
```

Comment:
good point 
does it really matter here? 
I think it doesn't, but if there's a quick way to not allow ".." might be an ok idea

I think we consider this to be open anyways, like if the user wanted to access the filesystem they can 
our threat model I think is more about stopping bad actors from executing code rather than limiting what they can do 

thoughts?